packages/core/src/app/address/StaticAddress.tsx

```diff
@@ -104,18 +113,26 @@ export function mapToStaticAddressProps(
 ): WithCheckoutStaticAddressProps | null {
     const {
         checkoutState: {
-            data: { getBillingCountries, getBillingAddressFields, getShippingAddressFields },
+            data: { getBillingCountries, getShippingCountries, getBillingAddressFields, getShippingAddressFields },
         },
+        checkoutService: {
+            loadBillingAddressFields, loadShippingAddressFields
+        }
     } = context;
 
     return {
-        countries: getBillingCountries(),
+        countries: type === AddressType.Billing
+            ? getBillingCountries()
+            : getShippingCountries(),
```

Comment:
Is this fixing another issue?
Looks like before this change we ignored the type and always returned billing countries. But billing countries are not necessarily the same as shipping countries. Is my understanding correct?

Reply:
Not sure if it is solving something else as well here. But the issue here was that for country name translations we need countries data from API and since Billing form was not yet rendered (in case where billing is same as shipping or refreshing checkout page after filling both forms) the API to get billing countries wasn't triggered.  

Yes, you are right, billing countries are not necessarily the same as shipping countries because we have different API's to fetch them.

Reply:
I guess I was thinking about this: https://github.com/bigcommerce/checkout-js/pull/1571
These two PRs are solving very similar problems 🤔

Reply:
Yes, apparently #1571 would solve this translation problem partially.

Reply:
@animesh1987 I don't know if we need two PRs here?
Maybe changes you're adding in your PR can be included in this PR instead. What do you think?

Reply:
Not entirely sure, as @richapatel1510 is adding support to `getShippingCountries` to StaticAddress. Where as in my PR I am preloading billing countries in shipping step.
So they might look similar but are different solutions. We can remove the call to loadCountries here but will need the change https://github.com/bigcommerce/checkout-js/pull/1568/files#diff-ded4d841a24b30f3c3ba48ec964a53a61045c8156846829a1aab4bb7ee986ed9R126 to get correct list of countries.

Reply:
But @snaderiBC to your point both bugs are result of the same symptom that we don't have the correct data loaded. I have added a detailed description in my PR, see if that makes sense.